### PR TITLE
Refactor the 'baseURI' property of the 'Code instrumentation' subsystem

### PR DESCRIPTION
**Pull request by LavrovArtem**
part of #1379

### CODE REVIEW

File: src/client/sandbox/native-methods.js [hunk 1]
```diff
@@ -307,6 +306,12 @@ class NativeMethods {
         if (cssStyleSheetHrefDescriptor)
             this.cssStyleSheetHrefGetter = cssStyleSheetHrefDescriptor.get;
 
+        const nodeBaseURIDescriptor = win.Object.getOwnPropertyDescriptor(win.Node.prototype, 'baseURI');
+
+        // NOTE: only for IE11
```
Comment (miherlosev) [inline src/client/sandbox/native-methods.js]: It's difficult to understand by your comment what only IE11 has `baseURI` or only `IE11` does not have this property.
Let's change to `// NOTE: IE11 does not have the 'window.Node.prototype.baseURI' property`

Comment (LavrovArtem) [reply src/client/sandbox/native-methods.js]: `// NOTE: IE11 does not have the 'baseURI' property` 
I think it better)))

And I will replace simular comments into current method.

Comment (miherlosev) [reply src/client/sandbox/native-methods.js]: Ok. Rebase this PR

**Comment by LavrovArtem** [comment]
@testcafe-build-bot \retest

**Pull request merged**